src/main/java/net/masterthought/cucumber/Configuration.java

```diff
@@ -310,4 +307,20 @@ public void setNotFailingStatuses(Set<Status> notFailingStatuses) {
             this.notFailingStatuses = notFailingStatuses;
         }
     }
+
+    public void setQualifier(@NonNull String jsonFile, @NonNull String qualifier) {
+        qualifiers.put(jsonFile, qualifier);
+    }
+
+    public String getQualifier(@NonNull String jsonFile) {
```

Comment:
`jsonFile` -> `jsonFileName` to make it clear it does not contain extension?

like `private String extractQualifier(String jsonFileName)`

Reply:
Done